### DISCUSSION TITLE
[Question]: How can custom plugins be displayed within the assistant plugin system

### DISCUSSION THREAD

**Discussion by coastli** [danny-avila/LibreChat#1906] [Q&A] [answered]
### What is your question?

How can custom plugins be displayed within the assistant plugin system?

### More Details

How can custom plugins be displayed within the assistant plugin system

### Screenshots

<img width="1263" alt="截屏2024-02-27 10 47 48" src="https://github.com/danny-avila/LibreChat/assets/3095550/f5bf0660-0318-4a86-9101-8fccf9aeebc0">
<img width="1265" alt="截屏2024-02-27 10 45 44" src="https://github.com/danny-avila/LibreChat/assets/3095550/f4c2d581-b0cd-4a4b-b247-ada87090e3e9">

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment, marked as answer]
I will make a guide on this soon. At the moment, it's pretty easy but not very intuitive. I want to make it as painless as possible on the dev side first.
